src/packages/address/elevatorRender.tsx

```
@@ -0,0 +1,380 @@
+              <div className={`${classPrefix}-selected-border`}>-</div>
+            ) : null}
+          </>
+        ))}
+      </div>
+    )
+  }
+
+  const renderHotArea = () => {
+    return (
+      <>
+        <div className={`${classPrefix}-title`}>热门城市</div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)